CorsixTH/Src/th_sound.cpp

```diff
@@ -26,49 +26,49 @@ SOFTWARE.
 #include <new>
 #include <cstring>
 
-THSoundArchive::THSoundArchive()
+th_sound_archive::th_sound_archive()
 {
-    m_pFiles = nullptr;
-    m_pData = nullptr;
+    sound_files = nullptr;
+    data = nullptr;
 }
 
-THSoundArchive::~THSoundArchive()
+th_sound_archive::~th_sound_archive()
 {
-    delete[] m_pData;
+    delete[] data;
 }
 
-bool THSoundArchive::loadFromTHFile(const uint8_t* pData, size_t iDataLength)
+bool th_sound_archive::load_from_th_file(const uint8_t* pData, size_t iDataLength)
 {
-    if(iDataLength < sizeof(uint32_t) + sizeof(th_header_t))
+    if(iDataLength < sizeof(uint32_t) + sizeof(th_header))
         return false;
 
     uint32_t iHeaderPosition = reinterpret_cast<const uint32_t*>(pData + iDataLength)[-1];
-    if(static_cast<size_t>(iHeaderPosition) >= iDataLength - sizeof(th_header_t))
+    if(static_cast<size_t>(iHeaderPosition) >= iDataLength - sizeof(th_header))
         return false;
 
-    m_oHeader = *reinterpret_cast<const th_header_t*>(pData + iHeaderPosition);
+    header = *reinterpret_cast<const th_header*>(pData + iHeaderPosition);
```

Comment:
`th_header` looks wrong, I guess this is the weak point of not using camel case for types. I'd go for renaming the class to something more specific, eg `sound_header` ?

Reply:
Maybe even `sound_dat_file_header`